argocd-notifications/README.md

```diff
@@ -0,0 +1,81 @@
+# Argocd-Notifications
+
+Docs: https://argocd-notifications.readthedocs.io/en/latest/
+
+The Docs are pretty sparse so I will try to provide all the work arounds and knowledge I gained by learning to deploy this.
+
+Also note that Argocd-notifications are currently deployed seperately, although that will be changed. [Argocd-Notifications has been migrated to argo-cd](https://github.com/argoproj/argo-cd/pull/7744), however is not available in any releases at the moment.
```

Comment:
Maybe change the wording a little bit.
I feel we should just inform the content which would be useful for the reader. 
and sentence like `I will try ` can be removed . what do you feel ?

Reply:
I agree, I think keeping it in the third person is best. Making the changes.